Source/Core/InputCommon/ControllerInterface/GameController/GameControllerDevice.h

```diff
@@ -0,0 +1,63 @@
+// Copyright 2010 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include <GameController/GameController.h>
+
+#include "InputCommon/ControllerInterface/CoreDevice.h"
+#include "InputCommon/ControllerInterface/ControllerInterface.h"
+
+namespace ciface::GameController
+{
+
+class Controller : public Core::Device
+{
+private:
+  class Button : public Input
+  {
+  public:
+    Button(GCControllerButtonInput* item) : m_item(item) {}
+    std::string GetName() const { return std::string([m_item.aliases.allObjects[0] UTF8String]); }
+    ControlState GetState() const override;
+
+  private:
+    const GCControllerButtonInput* m_item;
+  };
+
+  class Motion : public Input
+  {
+  public:
+    enum button
+    {
+      accel_x = 0,
```

Comment:
```suggestion
      AccelX = 0,
```

ditto for similar formatting on the rest of the entries.